q2_assembly/_utils.py

```diff
@@ -120,4 +120,4 @@ def _modify_links(fp: str):
 
 def _get_sample_from_path(fp):
     """Extracts sample name from a contig's file path."""
-    return os.path.basename(fp).split("_", maxsplit=1)[0]
+    return os.path.basename(fp.replace("_contigs.fa", ""))
```

Comment:
See my comment below for an explanation of why I'm making this suggestion. It additionally looks like I had messed up the call to `os.path.basename` here - that makes more sense to call on `fp` than the result of `fp.replace`. 

```suggestion
    return os.path.basename(fp).rsplit('_contigs.fa', maxsplit=1)[0]
```